src/main/resources/migrations.xml

```diff
@@ -82,4 +82,24 @@
 
         <addDefaultValue tableName="tokens" columnName="token_type" defaultValue="CARD"/>
     </changeSet>
+
+    <changeSet id="add type column to tokens table" author="">
+        <addColumn tableName="tokens">
+            <column name="type" type="varchar(36)">
+                <constraints nullable="true" unique="false"/>
+            </column>
+        </addColumn>
+    </changeSet>
+
+    <changeSet id="backfill existing tokens with API default value" author="">
+        <sql>
+            UPDATE tokens SET type = 'API';
+        </sql>
+    </changeSet>
+
+    <changeSet id="add not null constraint and default value to type in tokens table" author="">
+        <addNotNullConstraint tableName="tokens" columnName="type" />
```

Comment:
I think adding a `NOT NULL` constraint will lock the table while it goes and checks that all the existing rows aren’t `NULL`, so I’m not sure this is safe